dask_cuda/tests/test_spill.py

```diff
@@ -7,14 +7,19 @@
 from distributed.utils_test import gen_cluster, loop, gen_test  # noqa: F401
 from distributed.worker import Worker
 from distributed import Client, get_worker, wait
-from dask_cuda import LocalCUDACluster
+from dask_cuda import LocalCUDACluster, utils
 from dask_cuda.device_host_file import DeviceHostFile
 from zict.file import _safe_key as safe_key
 
 import dask
 import dask.array as da
-import cupy
-import cudf
+
+cupy = pytest.importorskip("cupy")
+cudf = pytest.importorskip("cudf")
```

Comment:
We should move this inside the respective tests. We can still test CuPy if cuDF isn't installed and vice-versa.

Reply:
Good point, Fixed